WcaOnRails/app/controllers/application_controller.rb

```diff
@@ -13,12 +13,19 @@ class ApplicationController < ActionController::Base
     ::NewRelic::Agent.add_custom_attributes(HTTP_USER_AGENT: request.user_agent)
   end
 
+  def self.locale_counts
+    @@locale_counts
+  end
+
   def set_locale
     # If the locale for the session is not set, we want to infer it from the following sources:
     #  - the current user preferred locale
     #  - the Accept-Language http header
     session[:locale] ||= current_user&.preferred_locale || http_accept_language.preferred_language_from(I18n.available_locales)
     I18n.locale = session[:locale] || I18n.default_locale
+
+    @@locale_counts ||= Hash.new(0)
+    @@locale_counts[I18n.locale] += 1
```

Comment:
We were loathe to that because you'd have to actually write it as this:

```ruby
# here
ApplicationController.locale_counts[I18n.locale] += 1
```

The `@@` syntax saves repeating `ApplicationController` and is a little less thinking overall.

> I guess that's fine to reset this whenever the app is reloaded?
  
Yeah. We want to keep it simple, and these numbers don't actually matter. We discussed putting it in the database, but it didn't seem worth it, these are just a sample. It's also important to remember that each unicorn worker is going to keep its own local copy of these numbers.